server/chat-plugins/chat-monitor.ts

```diff
@@ -473,7 +511,14 @@ export const commands: ChatCommands = {
 				if (Chat.monitors[list].punishment === 'EVASION') {
 					regex = constructEvasionRegex(word);
 				} else {
-					regex = new RegExp(Chat.monitors[list].punishment === 'SHORTENER' ? `\\b${word}` : word, filterTo ? 'ig' : 'i'); // eslint-disable-line no-unused-vars
+					regex = new RegExp(
+						Chat.monitors[list].punishment === 'SHORTENER' ?
+							`\\b${word}` :
+								word,
+						filterTo ?
+							'ig' :
+								'i'
+					); // eslint-disable-line no-unused-vars
```

Comment:
Please don't put everything on one line just because you can. :/

```ts
					regex = new RegExp(
						Chat.monitors[list].punishment === 'SHORTENER' ? `\\b${word}` : word,
						filterTo ? 'ig' : 'i'
					); // eslint-disable-line no-unused-vars
```

Reply:
force of habit sorry